src/styles/global.css

```
@@ -2745,41 +2818,38 @@ a.tag:hover {
+   Supersedes #271 (inactivity nudges + on-load settle). */
 @keyframes panel-pulse {
   0%, 100% { filter: none; }
   50%      { filter: brightness(1.08) saturate(1.05); }
 }
 
-.panel--settling {
-  animation: panel-settle var(--motion-plane) var(--ease-standard) both;
-  animation-delay: calc(var(--settle-index, 0) * var(--motion-stagger-step));
-  transform-origin: center center;
-  will-change: transform, opacity;
+@keyframes panel-pulse--neutral {
```

Stylelint (17.9.0)

[error] Expected keyframe name "panel-pulse--neutral" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)